_docs/021-getting-started-gke.md

```diff
@@ -68,17 +68,16 @@ If necessary change the default project.
 gcloud config set project $GCP_PROJECT_ID
 ```
 
-To list the available compute regions and zones:
+List the available compute regions and CPU quotas in each.
 
 ```sh
-gcloud compute zones list
+gcloud compute regions list
 ```
 
-To change the default region and zone:
+Choose a default region with at least 36 CPUS available, and set that to your default.
```

Comment:
if we set the default region shouldn't we also set the zone? otherwise we could end up with this:
```
$ gcloud config list compute/
[compute]
region = us-east1
zone = us-central1-c
```
maybe instead of changing the defaults, set an env var GCP_REGION and use that when creating the cluster instead of relying on the default set

Reply:
I removed the reference to zones because there does not appear to be a way get the quota by zone.
I have taken your suggestion of using a `GCP_REGION` env var, instead of setting the gcloud config.